ecmascript/parser/src/parser/typescript.rs

```diff
@@ -1687,11 +1698,13 @@ impl<'a, I: Tokens> Parser<'a, I> {
                 return self.parse_ts_import_type().map(TsType::from).map(Box::new);
             }
 
-            tok!("this") => {
+            tok!("this") | tok!("asserts") if is!("this") || peek_is!("this") => {
```

Comment:
How would you recommend checking for "asserts" here and above for `is!("asserts")`? This doesn't compile.

Reply:
You need to add "asserts" to tok! macro.